packages/edit-site/src/components/editor/index.js

```diff
@@ -200,6 +201,21 @@ function Editor() {
 											<SidebarComplementaryAreaFills />
 											<InterfaceSkeleton
 												labels={ interfaceLabels }
+												drawer={
+													<NavigationSidebar
+														content={
+															leftSidebarContent
+														}
+														isOpen={
+															isNavigationOpen
+														}
+														onNavigationToggle={ () =>
+															toggleLeftSidebarContent(
```

Comment:
(More of a pondering than a request)

Given that now the navigation sidebar is completely separated from the left sidebar, I think they should be handled by different states (e.g. `setLeftSidebar` and `setNavigationSidebar`).

With different states we could even have both sidebar visible at the same time.
I don't know if it's something we want, though, as we might end up with 3 sidebars open at the same time, which feels very cluttered to me.

Either way, we can make navigation and left sidebars "interdependent": when you open one, automatically close the other.

Reply:
I wanted to put the nav state into redux so I could use it in a random component, so i tried this out here: https://github.com/WordPress/gutenberg/pull/26003